kernel/src/main/java/org/kframework/parser/ParserUtils.java

```diff
@@ -199,8 +202,15 @@ private String loadDefinitionText(File definitionFile) {
 
         new CollectProductionsVisitor(kore, context).visit(def);
 
-        Map<String, List<org.kframework.kil.Module>> groupedModules = kilModules.stream()
-          .collect(Collectors.groupingBy(org.kframework.kil.Module::getName));
+        // Triple of moduleName, Source, Location
+        Map<String, List<Triple<String, Source, Location>>> groupedModules =
```

Comment:
we generally have been using scala tuples rather than apache commons lang3 where possible in new code. So please make this a Tuple3 instead.